cmd/openshift-tests/openshift-tests.go

```diff
@@ -526,6 +526,7 @@ func bindOptions(opt *runOptions, flags *pflag.FlagSet) {
 
 func bindTestOptions(opt *testginkgo.Options, flags *pflag.FlagSet) {
 	flags.BoolVar(&opt.DryRun, "dry-run", opt.DryRun, "Print the tests to run without executing them.")
+	flags.BoolVar(&opt.SkipInvariants, "skip-invariants", opt.SkipInvariants, "Skip invariant tests.")
```

Comment:
Title "Don't run invariant tests on CSI test suite" implies that this option will be `false` in `openshift/csi` suite, but I don't see any code there to do that.

Reply:
Old title, renamed.